gridstatus/ercot.py

```diff
@@ -1773,6 +1777,41 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    # @support_date_range("DAY_START") ??
+    def get_sced_system_lambda(self, date, verbose=False):
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+
+        doc = self._get_document(
+            report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+            date=date,
+            verbose=verbose,
+        )
+
+        def rename_func(df):
```

Comment:
does it make sense for this dataset to get parsed to an interval start and interval end? 

it seems to me like it would be fine to just leave the columns as`SCED Time Stamp` and `System Lambda` for now. if we do want to do parse to intervals, we'll need some logic like you see [here](https://github.com/kmax12/gridstatus/blob/main/gridstatus/ercot.py#L1143). Basically, this converts the sced time stamp to the 15 minute spp interval. 

in either case,  you'll have to update `read_doc` function or just create a new one, since it doesn't really handle this well. right now. 

We will face a similar question when we integrate the real time 5 minute LMPs (we currently only do SPP).

Reply:
I am so down to not  deal with interval ending if we don't need to 

IIRC this is an interval ending data type, but we can reach out to ERCOT to ask if need be

Reply:
let's just keep it as non interval ending for now and we can also enhance later